lib/github/repository.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module Github
+  # Class representing a Github Repository
+  class Repository
+    include Github::Pagination
+    include Github::Inspect
+
+    attr_accessor :access_token, :repo, :octokit_client, :github
+
+    # Creates a Github::Repository object with the github response attached
+    #
+    # @param repo [String] A GitHub repository
+    #
+    # @return [Github::Repository]
+    # @see https://docs.github.com/en/rest/reference/repos#get-a-repository
+    def initialize(access_token, repo)
+      @access_token = access_token
+      @repo = repo
+      @octokit_client = Octokit::Client.new(access_token: @access_token)
+      @github = octokit_client.repository(@repo)
+    end
+
+    # Get all repositories in an organization
+    #
+    # @param page [Integer] Page number
+    #
+    # @return [Sawyer::Resource] Repositories
+    # @see https://docs.github.com/en/rest/reference/repos#list-organization-repositories
+    def self.all(access_token, page = 1)
+      octokit_client = Octokit::Client.new(access_token: access_token)
+      response = {}
+      response[:repositories] = octokit_client.organization_repositories(GITHUB_ORGANIZATION, page: page)
+
+      response[:pages] = page_numbers(octokit_client)
+      response
+    end
+
+    # List all issues in a repository
+    #
+    # @param page [Integer] Page number
+    #
+    # @return [Sawyer::Resource] Issues
+    # @see https://docs.github.com/en/rest/reference/issues#list-repository-issues
+    def issues(page = 1)
+      Github::Issue.all(@access_token, @repo, page)
```

Comment:
Can @access_token and @repo here be changed to access_token and repo since they both have attr_accessors defined?